spec/ddoc.dd

```diff
@@ -513,7 +513,7 @@ $(P
 ------------------------------------
 
 $(P
-$(EM Enable support for backticks, tildes and other code languages with the `-preview=markdown` compiler flag.)
+$(EM Enable support for backticks, tildes and other code languages with the `-preview=markdown` compiler flag - enabled by default in 2.094.)
 )
```

Comment:
Why not simply remove this comment?

Reply:
Wasn't sure if the docs should cater towards older compilers

Reply:
No, they don't need to. We have doc archives for that.

Reply:
Done